pkg/sysfs/syspath_test.go

```diff
@@ -76,20 +78,24 @@ func TestGetParent(t *testing.T) {
 }
 
 func TestGetDeviceSysPath(t *testing.T) {
-	sysFSDirectoryPath = "/tmp/sys/"
-
+	tmp := sysFSDirectoryPath
+	sysFSDirectoryPath = filepath.Join(t.TempDir(), "sys/") + "/"
+	t.Cleanup(func() {
+		sysFSDirectoryPath = tmp
```

Comment:
q: How will this work?

Reply:
When will this be executed. Also it sets the `sysFSDirectoryPath` back to the actual `/sys`

Reply:
once the test is completed. just to reset the variable back to its value. It won't make any difference for now, but just added it in case not reverting back has some side effects in the future.